Project.toml

```diff
@@ -29,34 +29,33 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Showoff = "992d4aef-0814-514b-bc4d-f2e9a6c4116f"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
-StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-CategoricalArrays = "0.5"
-Colors = "0.9"
-Compat = "2"
-Compose = "0.7"
-Contour = "0.5"
-CoupledFields = "0.1"
-DataStructures = "0.15"
-Distributions = "0.18"
-DocStringExtensions = "0.7"
-Hexagons = "0.2"
-IndirectArrays = "0.5"
-IterTools = "1"
-JSON = "0.20"
-Juno = "0.7"
-KernelDensity = "0.5"
-Loess = "0.5"
-Measures = "0.3"
-Showoff = "0.2"
-StatsBase = "0.29.0"
-julia = "0.7, 1"
-CSV ="0.4"
-Cairo = "0.5"
-DataFrames = "0.17"
-RDatasets = "0.6"
-Unitful = "0.15"
+# known to work for these, but as it's not an exhaustive list, don't forcefully restrict
```

Comment:
For what it's worth, if you're going to get rid of the compat section but still want to document versions of packages that work, I'd delete all this and just go ahead and commit a Manifest.toml. That's what that file is for after all.